lib/Controller/Module.php

```diff
@@ -622,6 +622,29 @@ public function deleteWidgetForm($widgetId)
         ]);
     }
 
+    /**
+     * Delete Playlist Widget Form
+     * @param int $widgetId
+     * @throws XiboException
+     */
+    public function deletePlaylistWidgetForm($widgetId)
```

Comment:
Would it make more sense for this function to be in the Playlist Dashboard Controller?